Tests/GitHubReleases.tests.ps1

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubLabels.ps1 module
+#>
+
+# This is common test code setup logic for all Pester test files
+$moduleRootPath = Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path)
```

Comment:
I _just_ changed the way we commonly get this path in the other test files after this PR was submitted...

```suggestion
$moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
```

Reply:
So, I'm pretty new to git and even less experienced with github; so I've probably messed up these updates somehow. But ... I pushed up these changes to https://github.com/ucsb/ucsb-sa-powershell-powershellforgithub/commit/55d54d8a948088e83ab24f1247b854480d354bc3 and there's now an `outdated` flag on the files within this pull request.

Is this the right way to push up the changes? Is there a better way?